crates/mun/src/main.rs

```diff
@@ -107,11 +108,15 @@ fn compiler_options(matches: &ArgMatches) -> Result<mun_compiler::CompilerOption
     })
 }
 
-fn runtime_options(matches: &ArgMatches) -> Result<mun_runtime::RuntimeOptions, failure::Error> {
-    let delay: u64 = matches.value_of("delay").unwrap().parse()?;
+fn runtime_builder(matches: &ArgMatches) -> Result<RuntimeBuilder, failure::Error> {
```

Comment:
Why not return a Runtime from this function?

Reply:
Changed per your recommendation.